### PR TITLE
Abstract the SeekableStreamConverter

### PR DESCRIPTION
**Pull request by brandonpollett**
## Description
Describe the changes in this PR.

## Related issues
Addresses [AB#96436](https://microsofthealth.visualstudio.com/f8da5110-49b1-4e9f-9022-2f58b6124ff9/_workitems/edit/96436).

## Testing
Describe how this change was tested.

### CODE REVIEW

**Comment by jovinson-ms** [comment]
@brandonpollett is this ready for review?

File: src/Microsoft.Health.Dicom.WebUtilities/Microsoft.Health.Dicom.WebUtilities.csproj [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.WebUtilities/Microsoft.Health.Dicom.WebUtilities.csproj:4]: nit: the indents here are too large 😛

File: src/Microsoft.Health.Dicom.WebUtilities/Microsoft.Health.Dicom.WebUtilities.csproj [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        <ImplicitUsings>enable</ImplicitUsings>
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.WebUtilities/Microsoft.Health.Dicom.WebUtilities.csproj:5]: I don't really love `ImplicitUsings` as I think they hide information from developers, especially junior ones. I'd vote to remove this line

File: src/Microsoft.Health.Dicom.WebUtilities/SeekableStreamConverter.cs [hunk 3]
```diff
@@ -3,41 +3,34 @@
...
-    public SeekableStreamConverter(IHttpContextAccessor httpContextAccessor, ILogger<SeekableStreamConverter> logger)
+    protected SeekableStreamConverter(ILogger<SeekableStreamConverter> logger)
     {
-        _httpContextAccessor = EnsureArg.IsNotNull(httpContextAccessor, nameof(httpContextAccessor));
         _logger = EnsureArg.IsNotNull(logger, nameof(logger));
     }
 
     /// <inheritdoc />
-    [SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Caller will dipose of Stream.")]
+    [SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "Caller will dispose of Stream.")]
```
Comment (wsugarman) [inline src/Microsoft.Health.Dicom.WebUtilities/SeekableStreamConverter.cs:28]: Technically `RegisterForDispose` will take care of it

**Review by wsugarman — DISMISSED** [review]

**Review by wsugarman — APPROVED** [review]

**Pull request merged**